jerry-core/ecma/operations/ecma-comparison.c

```diff
@@ -31,6 +31,10 @@
  *
  * See also: ECMA-262 v5, 11.9.3
  *
+ * Note:
+ *      This function might raise an exception, so the
+ *      returned value must be freed with ecma_free_value.
```

Comment:
So... then the  `@return` entry below is incorrect. Please update it accordingly.

Reply:
I thought this would be an appropriate fix for the wrong `return` comment. Could you suggest something which makes it more clearer? Remove the `Note` and put the sentence into the `@return` block?

Reply:
We can keep the note and modify the return like this:
```
return true - if values are equal,
       false - if values are not equal,
       error - in case of any problems
```